extensions/nbextensions.py

```diff
@@ -36,15 +36,12 @@ def get(self):
 
         # Build a list of extensions from YAML file description
         # containing at least the following entries:
-        #   Type          - identifier
-        #   Compatibility - compatible notebook version, e.g. '4.x'
-        #   Name          - unique name of the extension
-        #   Description   - short explanation of the extension
-        #   Main          - main file that is loaded, typically 'main.js'
+        #   Type - identifier, must be either 'IPython Notebook Extension' or 'Jupyter Notebook Extension'
+        #   Name - unique name of the extension
+        #   Main - relative path to main file that is loaded, typically 'main.js'
         #
         extension_list = []
-        required_keys = (
-            'Type', 'Compatibility', 'Name', 'Main', 'Description')
+        required_keys = ('Type', 'Name', 'Main')
```

Comment:
Actually, to be honest, we could even dispense with requiring `Name`, using the URL of the main JavaScript file as the default